setup.py

```diff
@@ -14,6 +14,6 @@
       url=info.URL,
       license=info.LICENSE,
       packages=find_packages(),
-      setup_requires=['numpy>=1.12,<1.22'],
```

Comment:
Same change is required in requirements.txt

Reply:
Is it ok now?

Reply:
Yes